server/graphql/common/update.js

```diff
@@ -30,6 +30,8 @@ export async function createUpdate(_, args, req) {
     throw new Error('This collective does not exist');
   } else if (!req.remoteUser.isAdminOfCollective(collective)) {
     throw new Forbidden("You don't have sufficient permissions to create an update");
+  } else if (args.update?.isChangelog && !req.remoteUser.isRoot()) {
```

Comment:
:ice_cream: We already made sure the update exists at this point

```suggestion
  } else if (args.update.isChangelog && !req.remoteUser.isRoot()) {
```